eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -228,6 +228,10 @@ jobs:
       - src/mono/wasm/templates/*
       - src/mono/wasm/testassets/*
       - src/mono/wasm/Wasm.Build.Tests/*
+      - src/mono/wasi/build/*
+      - src/mono/wasi/runtime/*
+      - src/mono/wasi/testassets/*
+      - src/mono/wasi/Wasi.Build.Tests/*
       - ${{ parameters._const_paths._wasm_chrome }}
```

Comment:
Good catch!!
Add this to `wasm_specific_except_wbt_dbg` also. We will split up the wasm/wasi bits in a future PR, but for now they get triggered together.